composer.json

```diff
@@ -2,8 +2,8 @@
     "name": "doctrine/event-manager",
     "type": "library",
     "description": "Doctrine Event Manager component",
-    "keywords": ["eventmanager"],
-    "homepage": "https://www.doctrine-project.org",
+    "keywords": ["eventmanager", "eventdispatcher", "event"],
+    "homepage": "https://doctrine-project.org/projects/event-manager.html",
```

Comment:
FWIW, this redirects to `www.doctrine-project.org/...`, so we might as well keep the `www` prefix or change the redirects.

Reply:
Hmm where did I copy it from... 🤔 
PR? 👍